ee/clickhouse/queries/funnels/test/__snapshots__/test_funnel.ambr

```diff
@@ -318,3 +318,1370 @@
+                  if(isNotNull(latest_2)
+                     AND latest_2 <= latest_1 + INTERVAL 7 DAY, dateDiff('second', toDateTime(latest_1), toDateTime(latest_2)), NULL) step_2_conversion_time,
+                  prop
+           FROM
+             (SELECT aggregation_target,
+                     timestamp,
+                     step_0,
+                     latest_0,
+                     step_1,
+                     latest_1,
+                     step_2,
+                     min(latest_2) over (PARTITION by aggregation_target,
+                                                      prop
+                                         ORDER BY timestamp DESC ROWS BETWEEN UNBOUNDED PRECEDING AND 0 PRECEDING) latest_2 ,
+                                        prop
+              FROM
+                (SELECT aggregation_target,
+                        timestamp,
+                        step_0,
+                        latest_0,
+                        step_1,
+                        latest_1,
+                        step_2,
+                        if(latest_2 < latest_1, NULL, latest_2) as latest_2 ,
+                        prop
+                 FROM
+                   (SELECT aggregation_target,
+                           timestamp,
+                           step_0,
+                           latest_0,
+                           step_1,
+                           min(latest_1) over (PARTITION by aggregation_target,
+                                                            prop
+                                               ORDER BY timestamp DESC ROWS BETWEEN UNBOUNDED PRECEDING AND 0 PRECEDING) latest_1,
+                                              step_2,
+                                              min(latest_2) over (PARTITION by aggregation_target,
+                                                                               prop
+                                                                  ORDER BY timestamp DESC ROWS BETWEEN UNBOUNDED PRECEDING AND 0 PRECEDING) latest_2 ,
+                                                                 if(has(['finance', 'technology'], prop), prop, 'Other') as prop
+                    FROM
+                      (SELECT aggregation_target,
+                              timestamp,
+                              if(event = 'sign up', 1, 0) as step_0,
+                              if(step_0 = 1, timestamp, null) as latest_0,
+                              if(event = 'play movie', 1, 0) as step_1,
+                              if(step_1 = 1, timestamp, null) as latest_1,
+                              if(event = 'buy', 1, 0) as step_2,
+                              if(step_2 = 1, timestamp, null) as latest_2,
+                              trim(BOTH '"'
+                                   FROM JSONExtractRaw(group_properties_0, 'industry')) AS prop
```

Comment:
Q: Would the same occur with any breakdown? 

It'd be more efficient if we did this JSONExtract as deep as possible (e.g. for now in FunnelEventQuery) rather than deep here.

Non-blocking though.

Reply:
Yes, this is something I noticed as well, and it's true for all breakdowns. We should push these down for all breakdowns, but I didn't want to do it in this PR.

Reply:
:100: was just curious on the context. :) Thanks!